src/client/driver_command.hh

```diff
@@ -28,12 +29,13 @@ class DriverCommand : public Command {
                                  ui32 user_id) const override;
   String GetExecutable() const override;
   String RenderAllArgs() const override;
+
+  bool CanFillFlags() const override { return !!clang_; }
+
   inline bool FillFlags(base::proto::Flags* flags, const String& clang_path,
                         const String& clang_major_version) const override {
-    if (clang_) {
-      return clang_->FillFlags(flags, clang_path, clang_major_version);
-    }
-    return false;
+    CHECK(clang_);
```

Comment:
It's better to place `DCHECK(CanFillFlags())` - it won't run in release runtime, but will detect logical errors in tests.